### DISCUSSION TITLE
Setting up shoes on windows

### DISCUSSION THREAD

**Discussion by rohit9625** [scarpe-team/scarpe#132] [General]
Hello guy's,
I am unable to download shoes from the website it always says can't be reached.
Can you please help me with this?

**Comment by noahgibbs** [comment]
Original Shoes is gone, yeah. You can find source code for it, but not (so far as I know) pre-built versions. Scarpe is an attempt to rebuild Shoes, not the original Shoes code. I don't know of it having been tested on Windows, but it hasn't existed very long either.

**Reply by rohit9625** [reply]
So, how can I setup Scarpe in my windows?

**Reply by noahgibbs** [reply]
If you have Ruby working, you can try "gem install webview_ruby" to see if it works. If so, you can clone the scarpe repo and start running examples. If not, you'll probably want to look at Webview (https://github.com/webview/webview) and see if you can get it installed and working. It *should* support Windows, but we don't currently test Scarpe that way.

**Comment by rohit9625** [comment]
I tried that but it doesn't work. It through an error every time. Should I have to install Linux distros.

**Reply by noahgibbs** [reply]
The short version is: we don't really know, we don't test on Windows :-/

**Comment by rohit9625** [comment]
Okay, then I'll try this in some Linux distro.
